integreat_cms/cms/models/media/media_file.py

```diff
@@ -238,6 +244,7 @@ def search(cls, region, query):
             Q(name__icontains=query)
             | Q(alt_text__icontains=query)
             | Q(file__icontains=query),
+            Q(region=region, is_hidden=False) | Q(region__isnull=True),
```

Comment:
```suggestion
```
Here I cannot make an automated suggestion, but it's the same problem like before: Please do not just add this filter, but edit the existing one:
https://github.com/digitalfabrik/integreat-cms/blob/4e29fe4641a3eebab384fee6d81cfc8f4f608ada/integreat_cms/cms/models/media/media_file.py#L243